src/internal_events/prelude.rs

```diff
@@ -2,7 +2,6 @@
     feature = "sources-apache_metrics",
     feature = "sources-aws_ecs_metrics",
     feature = "sources-aws_kinesis_firehose",
-    feature = "sources-http-client",
```

Comment:
Does this line get removed or just corrected to `feature = "sources-http_client",` with an underscore?

Reply:
That's a good question. I had initially just corrected it, but since the feature before correction was _always_ false, it obviously wasn't necessary so I just removed it.